### PR TITLE
Update resource paths

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

File: gnomad/resources/grch38/gnomad.py [hunk 1]
```diff
@@ -395,7 +395,7 @@ def coverage_tsv_path(data_type: str, version: Optional[str] = None) -> str:
     if data_type == "exomes":
         if version is None:
             version = CURRENT_EXOME_RELEASE
```
Comment (mike-w-wilson) [inline gnomad/resources/grch38/gnomad.py:397]: Should this be updated to `CURRENT_EXOME_COVERAGE_RELEASE` to match genomes?

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]
Just one thing!

File: gnomad/resources/grch38/gnomad.py (hunk 1 above)
Comment (klaricch) [reply gnomad/resources/grch38/gnomad.py:397]: yes, thank you!

**Review by mike-w-wilson — APPROVED** [review]
:shipit:

**Comment by mike-w-wilson** [comment]
I'm not sure if its helpful, but to get around the error youre seeing in the check, I went back to python 3.8.6 and hail 02.1.20. I'm not entirely sure which fixed it but I think its python and hail is just importing from it?

**Comment by jkgoodrich** [comment]
I think [this](https://github.com/broadinstitute/gnomad_methods/pull/642/commits/4c2cf620b6924d1e2ec093f880406362f946292a) commit will fix that error

**Pull request merged**